src/SelfServe/SqlX/SqlX.tsx

```diff
@@ -339,7 +377,7 @@ export default class SqlX extends SelfServeBaseClass {
   instances: number;
 
   @Values({
-    labelTKey: "Cost",
+    labelTKey: "Approximate Cost Per Hour",
```

Comment:
Add entry for "Approximate Cost Per Hour" in SqlX.json file 

example 
{
...
...
"ApproximateCost": "Approximate Cost Per Hour",
...
...
}